common/rabbitmq/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
if the user keeps the default values you set, then no need to create this resources at all 
i think you need to put this resource under an if condition (if enableDetailedMetrics OR enablePerObjectMetrics )

Reply:
Done in latest commit